events/README.md

```diff
@@ -247,6 +249,41 @@ Type(s) of lane used or blocked `vehicle_blocked_lane_types` by the vehicle perf
 
 [Top][toc]
 
+
+### Payment Channel
+
+The payment channel describes the medium or platform used to pay for a curb
+event. This helps disambiguate a credit card payment made at a physical meter
+from a credit card payment made via a mobile app, for example.
+
+| Name              | Description                                            |
+| ----------------- | ------------------------------------------------------ |
+| `meter`           | User paid at a physical meter. |
+| `mobile_app`      | Paid via a mobile app. |
+| `mobile_app_clip` | Paid via a temporarily downloaded mobile app clip, not a full mobile app. |
+| `sms`             | Paid via text message. |
+| `website`         | User went to a standard website to pay, maybe directed by QR code. |
+| `other`           | Some payment channel not captured above (please submit a pull request!). |
+
+### Payment Method
+
+Strings used to indicate how a curb user paid for a curb event.
+
+| Name              | Description                                            |
+| ----------------- | ------------------------------------------------------ |
+| `cash`            | Bills or coins at a meter. |
+| `credit_card`     | Visa, Mastercard, etc at a meter. |
+| `smart_card`      | A specialized smart card. |
+| `membership_card` | A card used at a meter to pay via a corporate membership or loyalty program, etc. |
```

Comment:
`mobile_wallet` for example: Apple Pay, Google Pay, Samsung Pay, PayPal, Zelle, Venmo